### ISSUE TITLE
WebID profile should get checked first for solid:oidcIssuer

### ISSUE THREAD

**Issue by elf-pavlik** [solid/oidc-auth-manager#36]
follow up on https://github.com/solid/solid-auth-client/issues/107#issuecomment-460009651

`preferredProviderFor()` seems to check `/.well-known/openid-configuration` while to my understanding it should first check WebID Profile. WebID Profile can stay hosted on a server that also hosts OIDC Issuer but the profile can choose to specify a different issuer and that issuer should take precedence.

**Comment by dmitrizagidulin** [comment]
Agreed. (As mentioned in the comment thread, the current order of operations was done as an optimization, and it's not really needed.)
